### ISSUE TITLE
Wish: Minimize GHPython Editor

### ISSUE THREAD

**Issue by AndersDeleuran** [mcneel/ghpython#63]
Hi Giulio and Steve,

I often find the need to minimize the script editor when jumping back and forth between the Grasshopper canvas and Python code. This typically occurs when writing scripts which are longer than a couple of hundred lines and I want to "stay in the same place" in the script while operating the GH canvas. Having to either close the editor, drag it out of the way or manually drag the window size to its smallest size is rather clunky.

In such cases the option to minimize the script editor would be really great. I imagine this could work similarly to minimizing Grasshopper itself (double-clicking the title-bar).

Thanks for keeping up the great work on integrating Python :)

Best,

Anders

**Comment by piac** [comment]
Agreed! We need this. I really think this kind of minor suggestions are important!
So they are not so minor after all! :)

**Comment by AndersDeleuran** [comment]
Exactly :)

One other minor wish is to add support for the "windows key" + arrow up/down/left/right shortcuts for maximizing/minimizing/left/right windows. These are quite handy when quickly setting up GH/Rhino/GHPython viewports.

Thanks again,

Anders
